.github/workflows/validate-nginx-config.yml

```diff
@@ -0,0 +1,63 @@
+name: Validate Nginx Configuration
+on:
+  workflow_call:
+    inputs:
+      port:
+        default: 8080
+        description: 'Nginx container port number.'
+        required: true
+        type: number
+      realipheader:
+        default: 'X-Forwarded-For' # Ref: https://cloud.google.com/load-balancing/docs/https#target-proxies
+        description: 'Nginx real IP header.'
+        required: true
+        type: string
+      templatedir: # Ref: https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/entrypoint/20-envsubst-on-templates.sh#L32
+        description: 'Nginx configuration template directory. Usually have files with .template extension.'
+        required: true
+        type: string
+      verbose:
+        default: true
+        description: 'Toggle verbose output. By setting this to true, the job will print out the nginx version, nginx config files, and generated config files.'
+        required: false
+        type: boolean
+      workdir:
+        default: 'web'
+        description: 'Default working directory to run the job.'
+        required: true
+        type: string  
+
+jobs:
+  validate-nginx-config:
+    name: validate-nginx-config
+    runs-on: ubuntu-latest
+    env:
+      PORT: ${{ inputs.port }}
+      REAL_IP_HEADER: ${{ inputs.realipheader }}
```

Comment:
I assume this environment variable is needed for some specific nginx configuration.

Do we have more flexible way to pass environment variable? I think when this action usage grows, this environment variable list will also become bigger and bigger.

Reply:
I think I can remove it and we can set it as `env` in the job level instead

Reply:
https://github.com/reearth/reearth-cms/pull/1408/files#diff-df28d2857e9b13272bbe2262257362a4e8a096e5d2c29f465471779c6e397edcR9
@joshuagibeonm using latest commit, we can dynamically put the exported headers. But still with the limitation of supported `string` type from github input

Reply:
I will continue this discussion on PR comment